apps/condo/domains/miniapp/schema/B2BAppRole.js

```diff
@@ -106,8 +115,10 @@ const B2BAppRole = new GQLListSchema('B2BAppRole', {
                 const appId = resolvedData['app']
                 const roleId = resolvedData['role']
                 const role = await getById('OrganizationEmployeeRole', roleId)
+                if (!role || role.deletedAt) throw new GQLError(ERRORS.ROLE_NOT_FOUND, context)
```

Comment:
Is it possible to test throwing of this error?

Reply:
I could not check this particular error using test utilities. 
This error occurs at an earlier stage: "Unable to connect a OrganizationEmployee.role<OrganizationEmployeeRole>".  

A test for this has been added already